lib/locales/hu.ts

```diff
@@ -148,7 +148,7 @@ export const locale: Locale = {
   names: {
     //'205_blessed_martyrs_of_japan': '205 Blessed Martyrs of Japan',
     adalbert_of_prague_bishop: 'Szent Adalbert püspök és vértanú',
-    adalbert_of_prague_bishop_patron_of_poland: 'Szent Adalbert püspök és vértanú, Legyelország főpátrónája',
+    adalbert_of_prague_bishop_patron_of_poland: 'Szent Adalbert, püspök és vértanú, Legyelország főpátrónája',
```

Comment:
I think the comma after `Adalbert` should be removed. It is not present in [_Direktórium 2022_](http://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=97) and according to [what you said](https://github.com/romcal/romcal/pull/403#discussion_r1018231279) (based on the Hungarian orthography rules) it should be removed.

Reply:
Yes, I put some commas into the names and then I thought about it and I deleted them but I forgot to correct this one. 😄